sqjobs/tests/tests.py

```diff
@@ -0,0 +1,172 @@
+        loaddata_command.hide_empty = None
+        loaddata_command.loaddata(self.fixtures)
+
+
+class PeriodicJobsTests(DjangoTestCase):
+    beat = Beat(None, 10, True)
+    beat.register_job(Adder)
+
+    def test_get_expired_jobs(self):
+        assert len(self.beat.get_expired_jobs()) == 2
+
+    def test_add_delayed_job_info_and_programmed_date(self):
+        expired_job = PeriodicJob.objects.get(pk=2)
+        expired_job_kwargs = self.beat.get_job_kwargs(expired_job)
+        kwargs_with_delayed_info = self.beat.add_delayed_job_info(
+            expired_job, expired_job_kwargs
+        )
+        assert kwargs_with_delayed_info['num1'] == 1
+        assert kwargs_with_delayed_info['num2'] == 2
+        assert kwargs_with_delayed_info[PeriodicJob.PROGRAMMED_DATE] == expired_job.next_execution.astimezone(pytz.timezone(expired_job.timezone)).isoformat()
+        assert kwargs_with_delayed_info[PeriodicJob.DELAYED_JOB]
+
+    def test_add_programmed_date_but_not_delayed_job_info(self):
+        future_job = PeriodicJob.objects.get(pk=4)
+        future_job_kwargs = self.beat.get_job_kwargs(future_job)
+        kwargs_without_delayed_info = self.beat.add_delayed_job_info(
+            future_job, future_job_kwargs
+        )
+        assert kwargs_without_delayed_info['num1'] == 2
+        assert kwargs_without_delayed_info['num2'] == 1
+        assert kwargs_without_delayed_info[PeriodicJob.PROGRAMMED_DATE] == future_job.next_execution.astimezone(pytz.timezone(future_job.timezone)).isoformat()
+        assert PeriodicJob.DELAYED_JOB not in kwargs_without_delayed_info
+
+    def test_enqueue_jobs(self):
+        broker = Standard(None)
+        with patch.object(Standard, 'add_job') as mock_add_job:
+            beat = Beat(broker, 10, True)
+            beat.register_job(Adder)
+            beat.register_job(Divider)
+
+            expired_job = PeriodicJob.objects.get(pk=2)
+            expired_next_execution = expired_job.next_execution
+            expired_minute = expired_next_execution.minute
+            expired_second = expired_next_execution.second
+
+            future_job = PeriodicJob.objects.get(pk=4)
+            future_next_execution = future_job.next_execution
+            assert future_next_execution > datetime.now(pytz.timezone('UTC'))
+
+            beat.enqueue_next_jobs(beat.get_expired_jobs())
+
+            expired_job = PeriodicJob.objects.get(pk=2)
+            assert expired_minute == expired_job.next_execution.minute
+            assert expired_second == expired_job.next_execution.second
+            assert expired_job.next_execution > expired_next_execution
+            assert expired_job.next_execution > datetime.now(pytz.timezone('UTC'))
+
+            future_job = PeriodicJob.objects.get(pk=4)
+            assert future_job.next_execution == future_next_execution
+            assert future_job.next_execution > datetime.now(pytz.timezone('UTC'))
+
+        assert mock_add_job.called
```

Comment:
I would not only check `add_job` method was called, but the arguments it received.